app/src/main/java/com/globant/marvelmvvm/data/repository/MarvelRepository.kt

```diff
@@ -0,0 +1,30 @@
+package com.globant.marvelmvvm.data.repository
+
+import com.globant.marvelmvvm.data.entity.Character
+import com.globant.marvelmvvm.data.mapper.MarvelMapper
+import com.globant.marvelmvvm.data.service.api.MarvelApi
+import com.globant.marvelmvvm.data.service.requestgenerator.MarvelRequestGenerator
+import com.globant.marvelmvvm.util.Constants.Companion.NOT_FOUND
+import com.globant.marvelmvvm.util.Result
+
+class MarvelRepository {
```

Comment:
this should be MarvelService not repository

Reply:
Done!